buildlib/azure-pipelines.yml

```diff
@@ -136,6 +136,25 @@ stages:
               ./contrib/buildrpm.sh -s -t -b
             displayName: Build tarball
 
+      # Publish JUCX to maven cental
+      - job: publish_jucx
+        displayName: Publish JUCX SNAPSHOT artifact to maven central
+        container: centos7
```

Comment:
Why `centos` and not `fedora` like in the release pipeline?

Reply:
Had fails with gpg on fedora: https://askbot.fedoraproject.org/en/question/32052/gpg2-problem-with-the-agent-no-pinentry/

Reply:
How does it work in the release pipeline?

Reply:
Will try now on fedora, maybe unrelated issue.

Reply:
`gpg: signing failed: No pinentry`
Is it important to run release in fedora container?

Reply:
Or need to add to Feora container `dnf instal -y pinentry` and rebuild it.

Reply:
Let's rebuild the container. I'll do it and re-run checks for this PR.

Reply:
Added `pinentry` to fedora Dockerfile

Reply:
Please use `fedora:2` image in both regular and release pipeline. If it works, I'll make a PR for Dockerfile. And this PR will make the switch.

Reply:
[`Error response from daemon: manifest for fedora:2 not found: manifest unknown: manifest unknown`](https://dev.azure.com/ucfconsort/ucx/_build/results?buildId=3991&view=logs&jobId=d3e6da6b-dace-5955-5a3f-1d9faad0a965&j=d3e6da6b-dace-5955-5a3f-1d9faad0a965&t=ae668c22-7187-4394-963e-8c73c96ebb09 ) - should i update name in the Dockerfile

Reply:
Ok found a way to not use pinetry at all, so no new container is needed